vscode/src/test/suite/testController.test.ts

```diff
@@ -1083,4 +1083,88 @@ suite("TestController", () => {
       debug.dispose();
     });
   }).timeout(10000);
+
+  test("running a test with the coverage profile", async () => {
+    await withController(async (controller) => {
+      const uri = vscode.Uri.joinPath(workspaceUri, "test", "server_test.rb");
+      const testItem = (await controller.findTestItem(
+        "ServerTest::NestedTest#test_something",
+        uri,
+      ))!;
+
+      const fakeServerPath = path.join(
+        __dirname,
+        "..",
+        "..",
+        "..",
+        "src",
+        "test",
+        "suite",
+        "fakeTestServer.js",
+      );
+
+      // eslint-disable-next-line no-process-env
+      workspace.ruby.mergeComposedEnvironment(process.env as any);
```

Comment:
What's the reasoning here?

Reply:
Without invoking `ruby.activate()`, the env object is empty and then we can't even find NodeJS itself.

Here, I'm using our `mergeComposedEnvironment` just to add the process env itself, so that we can find node.